clouds/snowflake/modules/test/utils/GET_TABLENAME.test.js

```diff
@@ -0,0 +1,22 @@
+const { runQuery } = require('../../../common/test-utils');
+
+describe('_GET_TABLENAME should detect the correct TABLENAME from table identifier', () => {
```

Comment:
I propose something like:

```
const { runQuery } = require('../../../common/test-utils');

describe.each([
    'databasename.schemaname.tablename',
    'schemaname.tablename',
    'tablename'
])('_GET_TABLENAME should extract the table name from any valid table identifier', (tableIdentifier) => {
    test(`When identifier is ${tableIdentifier}: should return 'tablename'`, async () => {
        const query = `SELECT _GET_TABLENAME('${tableIdentifier}') AS result`;
        const rows = await runQuery(query);
        expect(rows[0].RESULT).toEqual('tablename');
    });
});
```
Which is less code and yields a slightly more readable output of:
```
Testing modules...
 PASS  test/utils/GET_TABLENAME.test.js
  _GET_TABLENAME should extract the table name from any valid table identifier
    ✓ When identifier is databasename.schemaname.tablename: should return 'tablename' (863 ms)
    ✓ When identifier is schemaname.tablename: should return 'tablename' (168 ms)
    ✓ When identifier is tablename: should return 'tablename' (216 ms)
```